tools/tracegen/TraceHeaderWriter.cpp

```diff
@@ -230,26 +231,30 @@ TraceHeaderWriter::tpTemplate(FILE *fd, unsigned int overhead, unsigned int test
 
 	if (test) {
 		/* Allow 7 digits for tracepoints + 1 for the null byte. (Millions of trace points are unlikely.) */
-		testMacro = (char *)Port::omrmem_calloc(1, (strlen(testMacroTemplate) + strlen(name) + strlen(module) + 8));
+		size_t bufLen = strlen(testMacroTemplate) + strlen(name) + strlen(module) + 8;
```

Comment:
While you're here, I see no reason not to allow room for the most digits possible in `unsigned int` (i.e. 7 -> 19) removing that assumption.

Reply:
Unsigned int requires 10 digits (4294967295) in 32 bit, and 20 digits (18446744073709551615) in the ILP64 model.
The author of the original code says `(Millions of trace points are unlikely.)` in the comment.

Reply:
Changed the code and the comment to allow 20 digits for `%u`.